src/controllers/voteController.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable indent */
+import voteService from '../services/VoteService';
+import PetitionService from '../services/PetitionServices';
+import Util from '../utils/Utils';
+
+const util = new Util();
+
+class voteController {
+  // eslint-disable-next-line consistent-return
+  static async addVote(req, res) {
+        if (!req.body.user_id || !req.body.vote || !req.params) {
+          util.setError(400, 'Please provide complete details');
+          return util.send(res);
+        }
+        // eslint-disable-next-line camelcase
+        const { vote, user_id } = req.body;
+        // userId get from token
+        const newId = req.params;
+        // eslint-disable-next-line camelcase
+        const petition_id = newId.id;
```

Comment:
Everything is good, but it seems you didn't destructure the id from the req.params and it caused you to end up writing redundant codes.

Reply:
let me take a look at it